notebooks/samples/explanations/ai-explanations-tabnet-algorithm.ipynb

```diff
@@ -0,0 +1,540 @@
+{
```

Comment:
<p>Can you add context here - explain that you're putting together all the mask values (from the entire provided sample prediction?) into a matrix to prepare it for visualization.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/182/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>